hooks/boot/30-upgrade.rb

```diff
@@ -4,3 +4,9 @@
   "Run the steps necessary for an upgrade such as migrations, rake tasks, etc.",
   :default => false
 )
+app_option(
+  '--disable-resolve-mismatches',
```

Comment:
Nitpick but for consistency sake, the Kafo convention for negative options would be `--no-resolve-mismatches`